Doc/using/configure.rst

```diff
@@ -287,6 +287,8 @@ General Options
 
    .. versionadded:: 3.11
 
+.. _free-threading-buid:
```

Comment:
I don't sure, but it's looks like unrelated to this PR. However, it's missing letter `l` in word `build` :)